### PR TITLE
Module 1

### PR DESCRIPTION
**Pull request by auchynnikau**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "presets": [
+    "@babel/preset-env",
+    "@babel/preset-typescript"
+  ]
+}
```
Comment (yanaSelin) [inline .babelrc:6]: no need in babel when ts in use

File: package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    "@babel/preset-env": "^7.13.10",
+    "@babel/preset-typescript": "^7.13.0",
+    "nodemon": "^2.0.7",
+    "ts-node": "^9.1.1"
+  },
+  "dependencies": {
+    "@types/node": "^14.14.35",
+    "csvtojson": "^2.0.10",
+    "dotenv": "^8.2.0",
+    "typescript": "^4.2.3"
```
Comment (yanaSelin) [inline package.json:34]: all type defs is dev deps, as well as typescript

File: src/task1.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  input: process.stdin,
+  output: process.stdout,
+});
+
+rl.on("close", () => {
+  process.exit(0);
+});
+
+const recursiveReadLine = () => {
+  rl.question("", (answer) => {
```
Comment (yanaSelin) [inline src/task1.ts:13]: would be great to print some instructions here

Comment (auchynnikau) [reply src/task1.ts]: Fixed.

File: package.json (hunk 2 above)
Comment (auchynnikau) [reply package.json:27]: Fixed.

File: .babelrc (hunk 1 above)
Comment (auchynnikau) [reply .babelrc]: Fixed.

**Pull request merged**